amplivelist/amp-live-list.go

```diff
@@ -100,6 +114,18 @@ func renderAmpLiveListSample(w http.ResponseWriter, page Page) {
 	t.Execute(w, page)
 }
 
+func renderLiveBlogSample(w http.ResponseWriter, page Page) {
```

Comment:
renderLiveBlogSample and renderLiveBlogSamplePreview are duplicating behavior. Extract common behavior into a method.